workshops/Tkinter/README.md

```diff
@@ -0,0 +1,135 @@
+---
+name: 'Creating a UI with Tkinter.'
+description: 'We will use python tkinter graphics to make User Interfaces. Its a nice world.'
+author: '@bezlin6mechminerz'
+---
+
+# Final User Interface or UI.
+
+<img width="539" alt="Screenshot 2020-09-19 at 11 28 35 AM" src="https://user-images.githubusercontent.com/54895485/93660382-e47c0880-fa6b-11ea-8f9a-94f476991652.png">
+
+This will be the simple UI we will be making today. This is nice world we can even make cool softwares with this .
+Here is the [live_demo](https://repl.it/@bezlin/graphics#main.py) and [final_code](https://repl.it/@bezlin/graphics#main.py) (see src/app.js and public/app.css files).
+This workshop will take 50 minutes.
+![ezgif com-video-to-gif](https://user-images.githubusercontent.com/54895485/93660884-07f58200-fa71-11ea-8bbd-44b10c7624b5.gif)
+
+## Step 1 : SETUP
+
+We will be using Tkinter python library to develop the user Interface.
+
+### Setting up the environment for Tkinter
+
+[Repl.it](https://repl.it) is a simple yet powerful online IDE, Editor, Compiler, Interpreter, and REPL. Code, compile, run, and host in 50+ programming languages.So it is super awesome.
+
+To get started, go to https://repl.it/languages/Tkinter. If you cant find it just type it in the type box. Your coding environment will be ready in a few seconds!
+<img width="1440" alt="Screenshot 2020-09-18 at 5 39 06 PM" src="https://user-images.githubusercontent.com/54895485/93595705-dbd8f300-f9d5-11ea-8bfa-a26e407adeaf.png">
+
+## Step 2: Know the basics of Tkinter
+
+Tkinter is a Python binding to the Tk GUI toolkit. It is the standard Python interface to the Tk GUI toolkit, and is Python's de facto standard GUI. Tkinter is included with standard Linux, Microsoft Windows and Mac OS X installs of Python. The name Tkinter comes from Tk interface.
+Why Tkinter?
+Tkinter provides a powerful object-oriented interface to the Tk GUI toolkit. Which can be customised and can look better.
+
+## Step 3: Code and explanation.
+
+<img width="961" alt="Screenshot 2020-09-18 at 6 06 14 PM" src="https://user-images.githubusercontent.com/54895485/93598005-a6ce9f80-f9d9-11ea-86f6-60226b8a37f2.png">
```

Comment:
ok, I'm not going to go through line by line here... but some general advice:

format code like

```python
import tkinter as tk
print("HIIIII")
```

or like `print("hi!")`

don't use images

also do the code with them step by step, don't give the solution at the start, let them figure it out :))))

Reply:
ok

Reply:
how can we give codes like this in coloured format